Packs/HuaweiFW/ModelingRules/HuaweiFW/HuaweiFW.xif

```diff
@@ -0,0 +1,198 @@
+
+[RULE:header_fields]
```

Comment:
Since this has already been done, don't change it now.
In future cases, if we reach the review stage with this, I would expect it to be done-

- For mapping cases such as this, it will be best practice to invest more in the key/value field extraction in the Parsing Rule. That way in the Modeling Rule we are using JSON sugar and extractions are more efficient and cleaner. This is what the command was made for.

- When allocating the message to xdm.event.description - If its a direct allocation without any changes then doing something like _xdm.event.description = extract_log_description_ is the right thing. If you add any changes, better to use _object_create_  for the extract_log_description field and the additional info you added.